src/playlist/playlistfilter.h

```diff
@@ -50,7 +50,8 @@ class PlaylistFilter : public QSortFilterProxyModel {
 
   void SetFilterText(const QString &filter_text);
 
-  QString filter_text() const { return filter_text_; }
+  QString filter_text() const { return filter_text_; };
```

Comment:
Why the extra semi-colon after }?

Reply:
I probably assumed I deleted one by mistake, because I am not used to seeing inline function definitions. 
I will try to be more thorough from now on. Also regarding the general style.